packages/x-components/src/components/sliding-panel.vue

```diff
@@ -236,14 +239,13 @@
       position: absolute;
       transition: all ease-out 0.2s;
       z-index: 2; /* To overlay the design system gradient with z-index:1 */
+    }
+    .x-sliding-panel-button-left {
+      left: 0;
+    }
 
-      &-left {
-        left: 0;
-      }
-
-      &-right {
-        right: 0;
-      }
+    .x-sliding-panel-button-right {
+      right: 0;
     }
```

Comment:
Not really since the component should be able to work without the design system. So we need to also have those styles here to make it work in case the XDS is not used.

Reply:
Oh, I get it now, you're right!